test/WebJobs.Script.Tests/UtilityTests.cs

```diff
@@ -69,7 +69,10 @@ public class UtilityTests
             sw.Start();
             await Utility.DelayWithBackoffAsync(2, CancellationToken.None);
             sw.Stop();
-            Assert.True(sw.Elapsed >= TimeSpan.FromSeconds(2), $"Expected sw.Elapsed >= TimeSpan.FromSeconds(2); Actual: {sw.Elapsed.TotalMilliseconds}");
+
+            // Workaround annoying test failures such as "Expected sw.Elapsed >= TimeSpan.FromSeconds(2); Actual: 1999.4092" by waiting slightly less than 2 seconds
+            // Not sure what causes it, but either Task.Delay sometimes doesn't wait quite long enough or there is some clock skew.
+            Assert.True(sw.Elapsed >= TimeSpan.FromMilliseconds(1990), $"Expected sw.Elapsed >= TimeSpan.FromMilliseconds(1990); Actual: {sw.Elapsed.TotalMilliseconds}");
```

Comment:
Would using `Math.Round(sw.Elapsed.TotalSeconds, 1)` be sufficient here?

Reply:
I know that some of the other test failures I have looked at that involve rounding code have taken me longer to understand what is exactly going on.